js/type_directives.ts

```diff
@@ -19,6 +18,12 @@ export function getMappedModuleName(
     return moduleName;
   }
   if (moduleName in typeDirectives) {
+    // @ts-ignore
```

Comment:
What is the typescript error hear we are ignoring... I hate to ignore anything.

Reply:
Also, not sure why we need to log an error out here...  Seems like debug stuff that should be removed.

Reply:
Doh, yes, sorry...